sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpMessageConstant.java

```diff
@@ -11,7 +11,7 @@
  * Reserved well-known constants from AMQP protocol.
  *
  * @see <a href="https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#type-properties">
- * AMQP 1.0: Messaging Properties</a>
+ *      AMQP 1.0: Messaging Properties</a>
```

Comment:
nit: this spacing can be removed.

Reply:
Hmm, never seen this before, will look into what setting causes this